index.js

```diff
@@ -24,7 +22,7 @@ Toolkit.run(async (tools) => {
   tools.log.complete("NPM install run");
 
   const { code } = await require("@zeit/ncc")(
-    path.join(workspace, "index.js"),
+    path.join(tools.workspace, tools.inputs.entrypoint),
```

Comment:
@JasonEtco Would `entrypoint` in this context be the entrypoint for the running action (`auto-compile-node`) or the repo that the workflow is running in?

Reply:
It’d be the input from the workflow file! Check out the README change for an example. It’s just a string that’s passed in the workflow.

Reply:
Ah, I think I've worked it out in a test run. Will add a comment below